blocks_verification.md

```diff
@@ -106,9 +106,22 @@ Verification:
 
 Server side:
 
+- The author must be a member of the group
 - The previous group block must be the hash of the last group modification of the group referenced by group_id
 - The keys in encrypted_group_private_encryption_keys_for_users.public_user_encryption_key should be non-obsolete keys
 
+### UserGroupRemoval
+
+This block is only verified server-side.
```

Comment:
should we call that a block ? since it is not part of the trustchain, should we call it something else like a message ?

Reply:
It has a block format, with the author, signature and everything, I think it's more appropriate to call it a block.